android/app/src/main/java/com/comapeo/MainApplication.kt

```diff
@@ -23,10 +23,7 @@ class MainApplication : Application(), ReactApplication {
             object : DefaultReactNativeHost(this) {
                 override fun getPackages(): List<ReactPackage> =
                     PackageList(this).packages.apply {
-                        // Packages that cannot be autolinked yet can be added manually
-                        // here,
-                        // for example:
-                        // add(MyReactNativePackage())
+                        add(WifiPackage())
```

Comment:
See the relevant [React Native docs](https://reactnative.dev/docs/next/native-modules-android?android-language=kotlin#register-the-module-android-specific).